taqueria-vscode-extension/src/test/suite/utils/utils.ts

```diff
@@ -1,3 +1,27 @@
+import retry from 'async-retry';
+import fsPromises from 'fs/promises';
+
 export function sleep(ms: number): Promise<void> {
 	return new Promise<void>(resolve => setTimeout(resolve, ms));
 }
+
+export async function checkFolderExistsWithTimeout(filePath: string) {
```

Comment:
We already have another copy of this function in `taqueria-vscode-extension/src/test/suite/utils/utils.ts`

Reply:
@AlirezaHaghshenas Yeas, and this is it. Sorry, I am not getting a bit this comment. It is the same file.